.gitignore

```diff
@@ -1,4 +1,8 @@
 
+# npm node_modules
+package-lock.json
```

Comment:
Hi [nit picking|optional], I was just glancing over this.
 its a practice to commit the lock file also. all the projects do commit their lock file also. wdyt ?

Reply:
Good spot, you're right. 👍